src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8356,6 +8659,1213 @@ .maxstack  2
+
+            var ctor0 = compilation0.GetMember<MethodSymbol>("C..ctor");
+            var ctor1 = compilation1.GetMember<MethodSymbol>("C..ctor");
+            var ctor2 = compilation2.GetMember<MethodSymbol>("C..ctor");
+
+            var v0 = CompileAndVerify(compilation0);
+            v0.VerifyIL("C..ctor", @"
+{
+  // Code size       33 (0x21)
+  .maxstack  3
+  .locals init (int V_0, //x
+                int V_1) //y
+  IL_0000:  ldarg.0
+  IL_0001:  ldloca.s   V_0
+  IL_0003:  call       ""int C.M(out int)""
+  IL_0008:  ldloc.0
+  IL_0009:  add
+  IL_000a:  ldloca.s   V_1
+  IL_000c:  call       ""int C.M(out int)""
+  IL_0011:  add
+  IL_0012:  call       ""Base..ctor(int)""
+  IL_0017:  nop
+  IL_0018:  nop
+  IL_0019:  ldloc.1
+  IL_001a:  call       ""void System.Console.Write(int)""
+  IL_001f:  nop
+  IL_0020:  ret
+}
+");
+
+            var md0 = ModuleMetadata.CreateFromImage(v0.EmittedAssemblyData);
+
+            var generation0 = EmitBaseline.CreateInitialBaseline(md0, v0.CreateSymReader().GetEncMethodDebugInfo);
+            var diff1 = compilation1.EmitDifference(
+                generation0,
+                ImmutableArray.Create(
+                    new SemanticEdit(SemanticEditKind.Update, ctor0, ctor1, GetSyntaxMapFromMarkers(source0, source1), preserveLocalVariables: true)));
+
+            diff1.VerifyIL("C..ctor", @"
+{
+  // Code size       18 (0x12)
+  .maxstack  3
+  .locals init (int V_0, //x
+                [int] V_1)
+  IL_0000:  ldarg.0
+  IL_0001:  ldloca.s   V_0
+  IL_0003:  call       ""int C.M(out int)""
+  IL_0008:  ldloc.0
+  IL_0009:  add
+  IL_000a:  call       ""Base..ctor(int)""
+  IL_000f:  nop
+  IL_0010:  nop
+  IL_0011:  ret
+}
+");
+
+            var diff2 = compilation2.EmitDifference(
+                diff1.NextGeneration,
+                ImmutableArray.Create(
+                    new SemanticEdit(SemanticEditKind.Update, ctor1, ctor2, GetSyntaxMapFromMarkers(source1, source0), preserveLocalVariables: true)));
+
+            diff2.VerifyIL("C..ctor", @"
+{
+  // Code size       33 (0x21)
+  .maxstack  3
+  .locals init (int V_0, //x
+                [int] V_1,
+                int V_2) //y
+  IL_0000:  ldarg.0
+  IL_0001:  ldloca.s   V_0
+  IL_0003:  call       ""int C.M(out int)""
+  IL_0008:  ldloc.0
+  IL_0009:  add
+  IL_000a:  ldloca.s   V_2
+  IL_000c:  call       ""int C.M(out int)""
+  IL_0011:  add
+  IL_0012:  call       ""Base..ctor(int)""
+  IL_0017:  nop
+  IL_0018:  nop
+  IL_0019:  ldloc.2
+  IL_001a:  call       ""void System.Console.Write(int)""
+  IL_001f:  nop
+  IL_0020:  ret
+}
+");
+        }
+
+        [Fact]
+        public void OutVar_InConstructorInitializer_WithLambda()
+        {
+            var baseClass = "public class Base { public Base(int x) { } }";
+
+            var source0 = MarkedSource(@"
+public class C : Base
+{
+    <N:0>public C() : base(M(out int <N:1>x</N:1>) + M2(<N:2>() => x + 1</N:2>)) { }</N:0>
```

Comment:
><N:0> [](start = 4, length = 5)

This is concerning - that we need to match scope syntax that is outside of the method body.